crates/ruff_python_formatter/src/snapshots/ruff_python_formatter__tests__black_test__expression_py.snap

```diff
@@ -659,17 +678,24 @@ last_call()
  )
 -Ø = set()
 -authors.łukasz.say_thanks()
--mapping = {
--    A: 0.25 * (10.0 / 12),
--    B: 0.1 * (10.0 / 12),
--    C: 0.1 * (10.0 / 12),
--    D: 0.1 * (10.0 / 12),
--}
 +result = NOT_IMPLEMENTED_call()
 +result = NOT_IMPLEMENTED_call()
 +Ø = NOT_IMPLEMENTED_call()
 +NOT_IMPLEMENTED_call()
-+mapping = {NOT_IMPLEMENTED_dict_key: NOT_IMPLEMENTED_dict_value}
+ mapping = {
+-    A: 0.25 * (10.0 / 12),
+-    B: 0.1 * (10.0 / 12),
+-    C: 0.1 * (10.0 / 12),
+-    D: 0.1 * (10.0 / 12),
++    A: 0.25
```

Comment:
this seems wrong, but not sure if it's an issue with dict formatting or the expressions for the values (or how those interact?)

Reply:
think i figured this out: need `group`